dnsmasq_interface.c

```diff
@@ -543,8 +543,6 @@ void FTL_cache(unsigned int flags, char *name, struct all_addr *addr, char *arg,
 				requesttype = QUERY_GRAVITY;
 			else if(arg != NULL && strstr(arg, "/black.list") != NULL)
 				requesttype = QUERY_BLACKLIST;
-			else if(flags & F_NXDOMAIN)
-				requesttype = QUERY_GRAVITY;
```

Comment:
Was this supposed to be removed?

Reply:
Yes, while revisiting this part, I saw that this is a situation that cannot occur.